embedded/no-allocator/src/main.rs

```diff
@@ -20,20 +20,21 @@ use panic_halt as _;
 fn main() -> ! {
     let mut encoded = ArrayString::<30>::new();
 
-    let base32 = [0x00u8, 0x01, 0x02].iter().copied().bytes_to_fes().collect::<ArrayVec<u5, 30>>();
+    let data = [0x00u8, 0x01, 0x02];
+    let hrp = Hrp::parse("bech32").expect("failed to parse hrp");
 
-    let hrp = Hrp::parse("bech32").unwrap();
-
-    bech32::encode_to_fmt_anycase(&mut encoded, hrp, &base32, Variant::Bech32).unwrap().unwrap();
+    bech32::encode_to_fmt::<Bech32, ArrayString<30>>(&mut encoded, hrp, &data)
```

Comment:
In 4da0a7b9784a7fe92db39a7d645d4bee97999d4d:

Could use `<Bech32, _>` rather than repeating `ArrayString<30>`.

Reply:
Done, thanks.